### ISSUE TITLE
Proguard file

### ISSUE THREAD

**Issue by jonandersen** [plaid/plaid-link-android#15]
Could you provide a working Proguard file? When running the SDK through Proguard it removes some of the required methods.

**Comment by amytang0** [comment]
Hi Jon,
```
-keep public class Plaid { public protected *; }
-keep public class LinkActivity
-keep public class LinkOAuthActivity
-keep public class PlaidAnalytics
-keep public class com.plaid.linkbase.models.* { public protected *; }
-keep public class com.plaid.plog.LogLevel { public protected *; }
-keep public class com.plaid.ribs.android.RibActivity
-keep public class com.plaid.ribs.android.LifecycleActivity
```

These lines should be comprehensive. Please let me know if this works for you.

**Comment by jonandersen** [comment]
@amytang0  still getting an error, everything works fine in debug mode. 

```
 E/PlaidFragment$resultHandler: Plaid Error PlaidApiError(displayMessage=, errorCode=, errorMessage=null, errorType=null, linkExitMetadata=LinkExitMetadata(linkSessionId=null, institutionName=null, institutionId=null, requestId=null, status=null))
```

```
 ComposedException 2 :
    	java.lang.NullPointerException: Attempt to invoke virtual method 'i.n.g.e.b.i i.n.g.e.b.g.b()' on a null object reference
        at i.n.e.w.h$e.a(LinkRootInteractor.kt:165)
        at i.n.e.w.h$e.a(LinkRootInteractor.kt:47)
        at k.b.f0.e.e.h$a.onError(SingleDoOnEvent.java:68)
        at k.b.f0.e.e.o$a.run(SingleObserveOn.java:79)
        at k.b.b0.b.b$b.run(HandlerScheduler.java:124)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

```
2019-12-18 09:38:54.787 2578-2578/com.public.app.development E/b$a: java.lang.IllegalStateException: 
        at i.n.e.w.h$d.a(LinkRootInteractor.kt:148)
        at i.n.e.w.h$d.a(LinkRootInteractor.kt:47)
        at k.b.f0.e.e.h$a.onSuccess(SingleDoOnEvent.java:55)
        at k.b.f0.e.e.o$a.run(SingleObserveOn.java:81)
        at k.b.b0.b.b$b.run(HandlerScheduler.java:124)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```


I tested this in the sample project in this repo and it has the same issue

**Comment by amytang0** [comment]
Hi Jon, thanks for the alert. This should be fixed with release 0.3.2.
